### ISSUE TITLE
CRUD Notes MVP Review

### ISSUE THREAD

**Issue by colineckert** [colineckert/everbrain#9]
Ready for review is my CRUD notes MVP!

Users can create new notes within notebooks (or within the `All Notes` view, which defaults to adding the note to `notebooks[0]`), as well as update and save changes to notes in the text editor, and delete notes either from within the editor component or `NotebookIndex`. 

Since the update note feature went hand-in-hand with the rich text editor, I feel that that MVP is also ready for review, if not close, and I would appreciate any feedback on it as well. 

Thanks, Lina!

**Comment by linakherchi** [comment]
Thanks, Colin ! Great job ! 

1 comment on your notes MVP: It seems that when a note has a some content and it is added to the sidebar of notes, the notes content seems to overlap with the date in which it was created.

I have a couple of comments about the editor:

1. The subheading dropdown can have more padding-right ? 
2. Size fonts can go for smaller -> bigger or the other way around - For now they seem to be out of order
3. It would be great that the line breaks automatically after we reach the sides of the client's screen 

Let me know if you have questions

Lina

**Comment by colineckert** [comment]
Hi Lina! Thanks for the direction.

I've added more padding to the toolbar dropdowns, organized the size fonts (and made them numerical so as not to be confused with the header dropdowns), and fixed the editor so that the line breaks on the word automatically when we reach the sides of the client's screen. I also fixed the issue with the overlapping in the notes sidebar. 

The latest build is ready for you in Heroku. Let me know if these MVPs look good!

Thanks,
Colin
